main/integration-tests/README.md

```diff
@@ -1,66 +1,90 @@
+* **Test Project:** Create a default project for running integration tests. (*Note the projectId)
 
-### For CI/CD Pipeline
+* **AWS Budget:** Create a budget for the AWS account the test project is associated with.
 
-- Ensure the `isBuildServer` parameter value is set to `true` in your `main/integration-tests/config/settings/<STAGE>.yml` config file
-- Ensure the `awsProfile` value corresponds to the correct AWS account
-- Follow the steps mentioned in the `main/cicd/README.md` file to set up the CI/CD Pipeline
-- The integration test config file is automatically saved in your S3 `<namespace>-artifacts` bucket as part of this process, in the `integration-test` folder
-
-### For GitHub Pipeline
-
-- Create/Locate the integration test config file in your S3 `<namespace>-artifacts` bucket's `integration-test` folder
-- Ensure the `isBuildServer` parameter value is set to `true` in your config file
-- Ensure the following GitHub secrets are created in your target repository: 
-  - AWS_ACCESS_KEY_ID
-  - AWS_SECRET_ACCESS_KEY
-  - DEPLOYMENT_BUCKET (same as `<namespace>-artifacts`)
+#### Resources to create through AWS Console:
 
+* **Test Administrator password:** Create an AWS Parameter Store record to store Test Administrator password. Pick a parameter name of your choice. Use 'SecureString' for parameter type 'text' for data type.(*Note the parameter name)
 
-### For Debugging Tests against Deployed Application
+* **Cost Explorer:** Turn on Cost Explorer service, activate tags tags 'Env', 'Proj' anc 'createdBy'.
 
-- Ensure the `isBuildServer` parameter value is set to `false`
+### Config File
 
-### For Debugging Tests Locally (SLS Offline)
+Once test resources are created, create a config file `main/integration-tests/config/settings/<STAGE>.yml` follow the example `main/integration-tests/config/settings/example.yml`.
 
-- Ensure the `isBuildServer` parameter value is set to `false`
-- Ensure the `isLocal` parameter value is set to `true`. Verify the `localApiEndpoint` parameter value is accurate
-- Remember to launch sls offline
+Use Test Admin username, projectId, and Parameter Store name noted from the Test Rousrces session for field username, passwordPath and projectId in the config file.
+Use the same config values used in `/main/config/settings` for fields awsRegion, awsProfile, solutionName, envName, ane envType. 
 
 **Note**
 
-This file is unique from the rest of the `<STAGE>.yml` files created in the SDCs in that it does not gather serverless settings passed on from the hierarchies above (eg. from `main/config/settings/.defaults.yml`)
+This file is unique from other `<STAGE>.yml` files under other SDCs. It does not gather serverless settings passed on from the hierarchies above (eg. from `main/config/settings/.defaults.yml`)
+
+## Execution
 
-## Execution:
+Once test resources and config file are created, you can run the integration tests against the dev environment defined in the config file.
```

Comment:
This is regarding the "you can run the integration tests against the dev environment defined in the config file" part.

The tests could be run against _any_ environment (dev, prod, local, CI etc.); what do you think of removing the 'dev' part from this string?

Reply:
I was trying to enforce the idea of not run the integ test against the prod environment. I'll update it to "against the non-production environment"?